### ISSUE TITLE
SA1101 false positive

### ISSUE THREAD

**Issue by elyoh** [Visual-Stylecop/Visual-StyleCop#35]
The following line:
```C#
XmlNode node = ... ;
var property = node["properties"]?["property"];
```
incorrectly gives:
SA1011 : CSharp.Spacing : Invalid spacing around the closing square bracket.	

Build 4.7.58.3

**Comment by ptittof57** [comment]
Fixed in the next release
